src/util/bloom.rs

```diff
@@ -0,0 +1,1016 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
```

Comment:
https://app.codecov.io/gh/blockstack/stacks-blockchain/compare/2884/tree/src/util/bloom.rs

Code coverage is pretty good for this file. 

BitField::clear isn't tested and some error conditions aren't tested. Might as well add a test for clear while you're in the area, up to you about error conditions.